### ISSUE TITLE
How to make this work in macvim?

### ISSUE THREAD

**Issue by jessicajzhang03** [gillescastel/inkscape-figures#48]
I use macvim, which, from what I understand, is supposed to be very similar to vim. Almost everything about this tool works for me, except that when I hit control+F to create a file, Inkscape always seems to glitch out. In particular, when I hit control+F in insert mode, Inkscape will be launched, but then will quit itself before opening up anything. But I can still edit: When I hit control+F in command mode, I can select the file I want to edit, and the tool launches Inkscape just fine. I'm pretty new to all this, so I was wondering if anyone has advice on how to fix this. Thanks!

**Comment by lucas0358** [comment]
Try to run vim in your terminal. C-f would work. (it works for me at least)

**Comment by jessicajzhang03** [comment]
Yes, vim in terminal works, but I was wondering about macvim in particular. I guess I could try neovim or something else though
